src/main/java/de/retest/recheck/cli/subcommands/Commit.java

```diff
@@ -102,6 +128,39 @@ private void applyChanges( final Persistence<SutState> persistence, final SuiteC
 		}
 	}
 
+	private void checkForWarningMessage( final ReviewResult changeSets ) {
```

Comment:
It's review result, so why `changeSets`? Also the method not just checks for warnings, it logs them. So how about:

```suggestion
	private void logWarnings( final ReviewResult reviewResult ) {
```